packages/reporter-vue/src/header/ReporterHeader.vue

```diff
@@ -31,57 +31,63 @@
   </ReporterHeaderLayout>
 </template>
 
-<script lang="ts">
-// import { useStatsStore, useReporterStore } from "../store";
-import { useStatsStore, useReporterStore  } from '../store/reporter-store'
+<script lang="ts" setup>
+import { useStatsStore } from '../store/reporter-store'
+import type { TestsByState } from '../store/reporter-store'
```

Comment:
I think we can merge these two together - `import type` is only needed if the module has no runtime exports (eg, `export const`, `export function` etc, which isn't this case here.

Reply:
Y'know, I think that script setup isn't playing nice with resolving types. I'm gonna avoid it.

Reply:
I like the idea of script setup, happy to adopt if/when the tooling improves (maybe we need to make a PR 💪 )